modules/replicator/src/main/java/org/apache/ignite/internal/replicator/listener/ReplicaListener.java

```diff
@@ -35,18 +32,7 @@ public interface ReplicaListener {
      */
     CompletableFuture<?> invoke(ReplicaRequest request, String senderId);
 
-    /**
-     * Callback on becoming the primary replica.
-     *
-     * @param clusterNode Primary replica node.
-     */
-    default void onBecomePrimary(ClusterNode clusterNode) {
-        // No-op.
-    }
-
-    /**
-     * Callback on replica shutdown.
-     */
+    /** Callback on replica shutdown. */
```

Comment:
Same here

Reply:
We discussed it as a team and decided not to remove the changes.